### PR TITLE
MTL-1664 CASMINST-4421 converge NTP documenation

### PR DESCRIPTION
**Pull request by jacobsalmela**
## Summary and Scope

Backports documentation from current branches to help prevent NTP issues during deployment/uograde.

## Issues and Related PRs

* Relates to CASMINST-4421
* Backports #1249 
* Backports #1247 

## Testing

Tested in original PRs

## Risks and Mitigations

Low.

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: install/bootstrap_livecd_remote_iso.md [hunk 1]
```diff
@@ -346,6 +346,10 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
          ```
 
       1. Generate the system configuration.
+      > **`NOTE`** ensure you select a reachable NTP pool/server passed in via the `--ntp-pools`/`--ntp-servers` flags, respectively.  Adding an uncreachable server can cause clock skew as chrony tries to continually reach out to a server it can never reach.
```
Comment (manderson-hpe) [inline install/bootstrap_livecd_remote_iso.md:349]: Two spaces after period of first sentence.

File: install/bootstrap_livecd_usb.md [hunk 2]
```diff
@@ -305,6 +305,9 @@ Some files are needed for generating the configuration payload. See these topics
 
    1. Generate the system configuration.
 
+   > **`NOTE`** ensure you select a reachable NTP pool/server passed in via the `--ntp-pools`/`--ntp-servers` flags, respectively.  Adding an uncreachable server can cause clock skew as chrony tries to continually reach out to a server it can never reach.
```
Comment (manderson-hpe) [inline install/bootstrap_livecd_usb.md:308]: Two spaces after period of first sentence.

**Review by manderson-hpe — CHANGES_REQUESTED** [review]

File: install/bootstrap_livecd_remote_iso.md (hunk 1 above)
Comment (jacobsalmela) [reply install/bootstrap_livecd_remote_iso.md]: 🔥 fixed

File: install/bootstrap_livecd_usb.md (hunk 2 above)
Comment (jacobsalmela) [reply install/bootstrap_livecd_usb.md]: fixed

**Pull request merged**